packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx

```diff
@@ -49,6 +29,82 @@ const useTabsStyles = makeStyles()((theme) => ({
         },
         margin: '0 !important',
     },
+    tab: {
+        whiteSpace: 'nowrap',
+        background: 'transparent',
+        color: theme.palette.maskColor.second,
+        '&:hover': {
+            background: 'transparent',
+        },
+    },
+    tabActive: {
+        background: '#fff',
```

Comment:
Avoid using hex string color. It's a dead-code.

```suggestion
        backgroundColor: '#fff',
```